sentence.go

```diff
@@ -87,6 +92,9 @@ func parseSentence(raw string) (BaseSentence, error) {
 
 // parsePrefix takes the first field and splits it into a talker id and data type.
 func parsePrefix(s string) (string, string) {
+	if s == TypeGSensord {
+		return "S", s
```

Comment:
Why "S"?

Reply:
I had no idea what to use, I figured "S for sensor makes sense" but now I know that P is for proprietary, I probably would do that.

Reply:
You might want to just leave it blank.